fvtest/compilertest/tests/LogFileTest.cpp

```diff
@@ -0,0 +1,262 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2016, 2016
+ *
+ *  This program and the accompanying materials are made available
+ *  under the terms of the Eclipse Public License v1.0 and
+ *  Apache License v2.0 which accompanies this distribution.
+ *
+ *      The Eclipse Public License is available at
+ *      http://www.eclipse.org/legal/epl-v10.html
+ *
+ *      The Apache License v2.0 is available at
+ *      http://www.opensource.org/licenses/apache2.0.php
+ *
+ * Contributors:
+ *    Multiple authors (IBM Corp.) - initial implementation and documentation
+ *******************************************************************************/
+
+#include "tests/LogFileTest.hpp"
+
+#include <stdio.h>
+#include <string.h>
+#include <fstream>
+#include <string>
+#include <sstream>
+#include <unistd.h>
+#include <sys/stat.h>
+
+#include "tests/OMRTestEnv.hpp"
+#include "tests/OpCodesTest.hpp"
+
+TestCompiler::LogFileTest::LogFileTest()
+   {
+   // Don't use fork(), since that doesn't let us initialize the JIT
+   ::testing::FLAGS_gtest_death_test_style = "threadsafe";
+   }
+
+TestCompiler::LogFileTest::~LogFileTest()
+   {
+   // Remove all generated log files.
+   for(auto it = _logFiles.begin(); it != _logFiles.end(); ++it)
+      {
+      unlink(*it);
+      }
+   }
+
+void
+TestCompiler::LogFileTest::compileTests()
+   {
+   ::TestCompiler::OpCodesTest unaryTest;
+   unaryTest.compileUnaryTestMethods();
+   //unaryTest.invokeUnaryTests();
+   }
+
+/**
+ * Determine if a file exists.
+ *
+ * @param name The name of the file.
+ */
+bool
+TestCompiler::LogFileTest::fileExists(const char *name)
+   {
+   // TODO: use port library for file operations
```

Comment:
Good idea.  I'd rather see that done now than hidden behind a TODO that will never be addressed.  Granted, the LimitFileTest.cpp has the same TODO, but arguably that one should be addressed sooner rather than later (but not in this pull request).

However, you are already using C++ streams for file I/O (and for good reason) throughout this test.  An alternative that I'm not opposed to is to make consistent use of C++ streams for all I/O operations throughout this test case.